README.md

```diff
@@ -136,18 +94,20 @@
 - 💨 Ad-free and clean results
 - 🌟 and lots more...
 
-**[⬆️ Back to Top](#websurfx)**
+**[⬆️ Back to Top](#--)**
 
 # Installation and Testing 🛠️ 
 
-> For full setup instructions, see: [**Installation**](https://github.com/neon-mmd/websurfx/wiki/installation)
+> For full setup instructions, see: [**Installation**](./docs/installation.md)
+
+Before you can start building `websurfx`, you will need to have `Cargo` installed on your system. You can find the installation instructions [here](https://doc.rust-lang.org/cargo/getting-started/installation.html).
 
-To get started with Websurfx, clone the repository, edit the config file, which is located in the `websurfx`{.verbatim} directory, and install the Redis server by following the instructions located [here](https://redis.io/docs/getting-started/) and then run the websurfx server and redis server using the following commands:
+To get started with Websurfx, clone the repository, edit the config file, which is located in the `websurfx/` directory, and install the Redis server by following the instructions located [here](https://redis.io/docs/getting-started/) and then run the websurfx server and redis server using the following commands:
 
 ``` shell
 git clone https://github.com/neon-mmd/websurfx.git
 cd websurfx
-cargo build
+cargo build -r
 redis-server --port 8082 &
 ./target/debug/websurfx
```

Comment:
```suggestion
./target/release/websurfx
```
You may have forgotten to update the path of the executable file, if we build in release mode, the path should be `./target/release/websurfx`

Reply:
My bad, I didn't notice this at all :smile: but thanks for pointing it :slightly_smiling_face: .